pb/metrics.proto

```diff
@@ -30,6 +37,7 @@ message Metric {
     double double_value = 11;
     string string_value = 12;
   }
+  map<string, string> unindexed_labels = 20;
```

Comment:
I'd argue for indexed_labels and unindexed_labels. I don't like the idea of the Labels symbol in the struct suddenly meaning something else because it means I could forget to rename something and things would still compile. I'd rather force the compile error in that case.